### DISCUSSION TITLE
Add Custom Languages

### DISCUSSION THREAD

**Discussion by XielQs** [Monika-After-Story/MonikaModDev#8241] [general - translations]
# Custom Languages

Game Files Should Have One Lang Folder And Files Containing Languages ​​In It, User Should Be able to Add New Languages ​​to the Game by Editing These Files

### Warning : This Article Written With Google Translate !

**Comment by Dobby233Liu** [comment]
Ren'Py has the feature you want, with that feature doing translations should be easy, [this unofficial Japanese translation submod is a great example to how to do so](https://github.com/proudust/ddlc-mas-jp-patch), I think a dev can explain it to you
